pkg/chain/ethereum/tbtc.go

```diff
@@ -194,13 +195,43 @@ func newTbtcChain(
 		)
 	}
 
+	redemptionWatchtowerAddress, err := bridge.GetRedemptionWatchtower()
+	if err != nil {
+		return nil, fmt.Errorf(
+			"failed to get RedemptionWatchtower address from Bridge: [%v]",
+			err,
+		)
+	}
+
+	var redemptionWatchtower *tbtccontract.RedemptionWatchtower
+	if redemptionWatchtowerAddress != [20]byte{} {
```

Comment:
We are allowing the `RedemptionWatchtower` not to be set, unlike other contracts which must always be set.
Could you please add a brief explanation why we treat the `RedemptionWatchtower` differently?

Reply:
Sure! Please see: dcdcdec0